.github/workflows/docker-build.yaml

```
@@ -372,6 +372,48 @@ jobs:
+        runs-on: ubuntu-22.04
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v4
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Change first domain locale to CS
+                uses: mikefarah/yq@v4.45.1
+                with:
+                    cmd: yq e -i '( .domains[] | select(.id == 1) ).locale = "cs" | ( .domains[] | select(.id == 2) ).locale = "en"' ./project-base/app/config/domains.yaml
+            -   name: Change first domain currency to CZK
+                run: sed -i 's/if ($domainId === Domain::SECOND_DOMAIN_ID)/if ($domainId === Domain::FIRST_DOMAIN_ID)/' ./project-base/app/src/DataFixtures/Demo/SettingValueDataFixture.php
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2016:info:1:8: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)